src/pke/include/scheme/ckksrns/ckksrns-fhe.h

```diff
@@ -199,8 +199,10 @@ class FHECKKSRNS : public FHERNS {
         ar(cereal::base_class<FHERNS>(this));
     }
 
+    // To be deprecated; left for backwards compatibility
```

Comment:
just a reminder: we can add an attribute to produce a trace for deprecated functions